app/components/common/Sidebar.tsx

```diff
@@ -57,6 +57,7 @@ export const SidebarItem: FC<SidebarItemProps> = ({
 }) => (
   <li className={cn("py-2.5 post:px-5 laptop:px-0 first:pt-5 last:pb-5 w-full", className)}>
     <SheetClose asChild>
+      {/* FIXME: This component breaks in sidebar and I have no idea why */}
```

Comment:
This also needs further investigation. Because it's the only place where Link breaks navigation.

Reply:
Never mind, it seem to be part of the problem with Prims, I guess. At least it was resolved after I disabled CodeBlock plugin.